mysql-test/suite/galera/t/galera_insert_bulk.test

```diff
@@ -30,28 +36,46 @@ while ($count < 1000)
 
 --connection node_2
 
-# Disable bulk insert.
+# Disable bulk insert on this node.
 SET foreign_key_checks = 1;
 SET unique_checks = 1;
 
 # Insert a value out of the bulk insert range.
 INSERT INTO t1 VALUES (1001);
 
 --connection node_1
+
+# We need to trigger Galera-level certification conflict. For this:
+# - start applying single insert from the other node before bulk insert certifies
+# - certifying bulk insert will lead to the conflict
+# - keep applying single insert
+SET DEBUG_SYNC = 'wsrep_before_certification WAIT_FOR sync.wsrep_apply_cb_reached';
+SET DEBUG_SYNC = 'wsrep_after_certification SIGNAL signal.wsrep_apply_cb';
+
 --error ER_LOCK_DEADLOCK
 COMMIT;
 
 DROP TABLE t1;
+SET GLOBAL DEBUG_DBUG = '';
+SET DEBUG_SYNC = 'RESET';
 
 #
 # Make bulk insert succeed, but regular insert BF-abort.
 #
 
+--connection node_2
+# Delay applying of the bulk insert from the other node.
+SET GLOBAL DEBUG_DBUG = 'd,sync.wsrep_apply_cb';
+
 --connection node_1
 CREATE TABLE t1 (f1 INTEGER PRIMARY KEY) ENGINE=InnoDB;
 
```

Comment:
there is now effective sync point before applying, but TOI replication does not go through this sync point, so t1 will be created in node_2